### PR TITLE
Remote screen resolution

### PR DESCRIPTION
**Pull request by PaulHax**
Changes rendered frame size based on client canvas size.

### CODE REVIEW

File: packages/agave-renderer/itk_viewer_agave_renderer/renderer.py [hunk 1]
```diff
@@ -47,6 +47,14 @@ async def setup(self):
         self.agave = AgaveRendererMemoryRedraw()
         self.agave.set_resolution(self.width, self.height)
 
+    def set_size(self, width, height):
```
Comment (thewtex) [inline packages/agave-renderer/itk_viewer_agave_renderer/renderer.py:50]: set_size -> set_render_size

File: packages/agave-renderer/itk_viewer_agave_renderer/renderer.py [hunk 2]
```diff
@@ -68,15 +75,17 @@ async def update_renderer(self, events):
...
-                    r.density(payload)
                 case 'cameraPose':
                    eye = payload['eye']
                    r.eye(eye[0], eye[1], eye[2])
                    up = payload['up']
                    r.up(up[0], up[1], up[2])
-                   target = payload['target']
+                   target = payload['center']
                    r.target(target[0], target[1], target[2])
+                case 'size':
```
Comment (thewtex) [inline packages/agave-renderer/itk_viewer_agave_renderer/renderer.py:85]: renderSize

**Review by thewtex — CHANGES_REQUESTED** [review]
Nice!!

Minor naming suggestions inline

File: packages/agave-renderer/itk_viewer_agave_renderer/renderer.py (hunk 2 above)
Comment (PaulHax) [reply packages/agave-renderer/itk_viewer_agave_renderer/renderer.py]: changed!

Comment (thewtex) [reply packages/agave-renderer/itk_viewer_agave_renderer/renderer.py]: :100:

**Review by thewtex — APPROVED** [review]
:100:

**Pull request merged**